support.md

```diff
@@ -77,10 +77,12 @@ Only active, stand-alone Medicare Prescription Drug Plan (PDP) sponsors are elig
 {% endcapture %}
 
 {% capture a5HeadingContent %}
-How can we get additional data elements beyond what’s listed in the <a href="https://www.federalregister.gov/documents/2019/04/16/2019-06822/medicare-and-medicaid-programs-policy-and-technical-changes-to-the-medicare-advantage-medicare#page-15745">final rule</a>?
+How can we get additional data elements beyond what’s listed in the final rule?
 {% endcapture %}
 
 {% capture a5AccordionContent %}
+<p>What is the <a href="https://www.federalregister.gov/documents/2019/04/16/2019-06822/medicare-and-medicaid-programs-policy-and-technical-changes-to-the-medicare-advantage-medicare#page-15745" target="_blank">final rule?</a></p>
+
```

Comment:
I'm not _exactly_ sure this is preferred, but this fixes the issue of having a link inside a button

Reply:
This doesn't affect the UX at all so fine with me!